app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -116,7 +116,8 @@ fun MessageComposer(
                     )
                 }
                 messageComposerState.focusManager.clearFocus()
-                messageComposerState.toInactive(clearInput = true)
+                messageComposerState.setMessageTextValue(TextFieldValue(""))
+                messageComposerState.toInactive()
```

Comment:
Should we extract these 3 lines to a separate method that can be reused maybe by the `onSendEditButtonClicked` and the `onEditCancelButtonClicked` ?

Reply:
Good idea, I added a new method in `messageComposerState` 😃